schemas/process.yml

```diff
@@ -497,3 +497,17 @@
       type: long
       description: >
         The length of bytes skipped.
+
+    - name: origin_referrer_url
```

Comment:
For process, I think it might be better to use `executable.origin_referrer_url` and `executable.origin_url`, since this is a property of the executable file that was used to create the process, not of the process itself.

This is something that's being discussed in otel for a different change: https://github.com/open-telemetry/semantic-conventions/pull/1737. ECS doesn't currently follow this convention to refer to the source executable file all the time (for example with `process.name`, but I think the otel convention is better, and something we should try to do here as well.

Reply:
@mjwolf I agree with @AsuNa-jp regarding restriction we currently have with executable being existing leaf [node](https://www.elastic.co/guide/en/ecs/current/ecs-process.html#field-process-executable). We have discussed that probably we would need to make adjustments in ECS to accommodate Otel schema. But I think it shouldn’t be a problem for this PR unless you have other ideas for naming

Reply:
I didn't think of executable already being a keyword, and the problems that would introduce when I made this suggestion. I agree it's best to leave this as it is for now